neat_ml/cli.py

```diff
@@ -66,14 +69,22 @@ def run(config: str) -> None:
                 continue
 
             model: object = None
+            gmodels = (
+                grape.get_available_models_for_edge_prediction()["model_name"]
+            ).tolist()
+            gmodels = [mname.lower() for mname in gmodels]
             if classifier["classifier_name"].lower() == "neural network":
                 model = MLPModel(classifier, outdir=yhelp.outdir())
-            elif classifier["classifier_name"].lower() in [
-                "decision tree",
-                "logistic regression",
-                "random forest",
-            ]:
+            elif (
+                classifier["classifier_name"].lower() == "logistic regression"
```

Comment:
Do we want to create a separate section of constants at the top (or constants.py) where:
```
LOGISTIC_REGRESSION = "logistic regression"
```
and then apply here?

Reply:
Good idea - could even be a list of acceptable variations/synonyms